src/main/java/org/jenkinsci/plugins/p4/console/P4ConsoleAnnotator.java

```diff
@@ -34,6 +34,22 @@ private void push(MarkupText text) {
 		text.addMarkup(COMMAND.length(), head);
 
 		StringBuffer sb = new StringBuffer();
+
+		sb.append("<script>\n");
+		sb.append("function toggle(showHideDiv, switchTextDiv) {\n");
+		sb.append("\tvar tog = document.getElementById(showHideDiv);\n");
+		sb.append("\tvar text = document.getElementById(switchTextDiv);\n");
+		sb.append("\tif(tog.style.display == \"block\") {\n");
+		sb.append("    \ttog.style.display = \"none\";\n");
+		sb.append("\t\ttext.innerHTML = \"+\";\n");
+		sb.append("  \t}\n");
+		sb.append("\telse {\n");
+		sb.append("\t\ttog.style.display = \"block\";\n");
+		sb.append("\t\ttext.innerHTML = \"-\";\n");
+		sb.append("\t}\n");
+		sb.append("}\n");
+		sb.append("</script>");
+
```

Comment:
I'll try and add the JS to the other Jelly directories for the missing classes, feel free to open a new PR if you like.
Please remove the change from the PR and I will approve the changes to Validate.java.